tour.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+<title>Tour</title>
+</head>
+<body>
+<div class="header">
+	<div class="wrapper">
+		<div class="top-header">
+			<div class="navigation">
+				<ul class="navigation-buttons">
```

Comment:
you have two unnecessary wrappers - top-header and navigation - remove them and rename ul.navigation-buttons to navigation